crates/trippy-tui/src/frontend/columns.rs

```diff
@@ -426,6 +425,7 @@ mod tests {
     #[test_case(ColumnType::StdDev, "StDev")]
     #[test_case(ColumnType::Status, "Sts")]
     fn test_column_display_formatting(c: ColumnType, heading: &'static str) {
+        let _ = crate::locale::init(None);
```

Comment:
This is not gonna cut it, as system language is taken into account, so in an environment with e.g. `LANG=zh` it will break. The exact reason I wrote `init(Some("en"))`.